packages/components/src/button/test/index.tsx

```diff
@@ -139,16 +139,6 @@ describe( 'Button', () => {
 			expect( screen.getByRole( 'button' ) ).toHaveClass( 'gutenberg' );
 		} );
 
-		it( 'should render an additional WordPress prop of value awesome', () => {
-			render( <Button WordPress="awesome" /> );
```

Comment:
Now that `Button` is in TypeScript, it can't accept an arbitrary prop like `WordPress`

Reply:
Good idea, committed in https://github.com/WordPress/gutenberg/pull/46997/commits/35f8efb841a4567f5204edeaf9556d5c6ad3df1e